<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkersList</class>
 <widget class="QWidget" name="MarkersList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>516</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MarkersList</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QFrame {
    background-color: #37474f;
border-top: none;
}
QLabel {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(55, 71, 79, 255), stop:0.5 rgba(38, 50, 56, 255), stop:1 rgba(34, 42, 48, 255));
    border: none;
    border-bottom: 1px solid #191919;
    color: #b0bec5;
    font: 600 16px &quot;Open Sans&quot;;
}

QSpinBox {
    border-radius: 2px;
}

EntityWidget  {
  background-color: #37474f;
  border: none;
}

QAbstractButton {
    font: 13px &quot;Open Sans Semibold&quot;;
    color: #fafafa;
}

QCheckBox::indicator {
    width: 12px;
    height: 12px;
    border-radius: 2px;
    border: 1px solid #191919;
    padding: -1px;
    subcontrol-position: left;
top: 1px;
}

QAbstractButton::indicator:!checked {
    background-color: rgba(250, 250, 250, 225);

}
QCheckBox::indicator:checked {
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(34, 42, 48, 255), stop:0.5 rgba(69, 90, 100, 255), stop:1 rgba(69, 90, 100, 255));
	image: url(:/textures/Common/icons/icon_check_shadowed.png);
}
QCheckBox::indicator:!checked:hover {
    background-color: rgba(250, 250, 250, 155);
}
QCheckBox::indicator:!checked:pressed {
border: 1px solid #191919;
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(132, 136, 148, 122), stop:0.5 rgba(150, 160, 165, 122), stop:1 rgba(150, 160, 165, 122));
	image: url(:/textures/Common/icons/icon_check_shadowed_transparent.png);
}
QCheckBox::indicator:checked:hover {
     background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(34, 42, 48, 200), stop:0.5 rgba(69, 90, 100, 200), stop:1 rgba(69, 90, 100, 200));
image: url(:/textures/Common/icons/icon_check_shadowed_transparent.png);
}
QCheckBox::indicator:checked:pressed {
border: 1px solid #191919;
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(132, 136, 148, 122), stop:0.5 rgba(150, 160, 165, 122), stop:1 rgba(150, 160, 165, 122));
	image: url(:/textures/Common/icons/icon_check_shadowed_transparent.png);
}
QAbstractButton::indicator:indeterminate {
    width: 12px;
height: 12px;
border: 2px inset #191919;
border-bottom: 2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(25, 25, 25, 255), stop:0.495 rgba(25, 25, 25, 255), stop:0.505 rgba(69, 90, 100, 255), stop:1 rgba(69, 90, 100, 255));
padding: -1px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(121, 121, 121, 255), stop:0.95 rgba(199, 199, 199, 255), stop:1 rgba(250, 250, 250, 255));
background-clip: padding;
background-origin: padding;
}
QAbstractButton {spacing: 4px;}
QRadioButton {spacing: 2px;}
QRadioButton::indicator {
    width: 12px;
    height: 12px;

    border: 1px solid #37474f;;
	background-image: url(:/textures/Common/buttons/bttn_radio.png);
	background-origin: margin;
}

QRadioButton::indicator:checked {
background-position: bottom left;
}
QRadioButton::indicator:checked:hover {
background-position: bottom center;
}
QRadioButton::indicator:checked:pressed {
background-position: bottom right;
}
QRadioButton::indicator:!checked:hover {
background-position: top center;
}
QRadioButton::indicator:!checked:pressed {
background-position: top right;
}
QRadioButton::indicator:checked:disabled  {
    background-position: center;
}
QRadioButton::indicator:!checked:disabled  {
    background-position: center left;
}
QAbstractButton:disabled {
    color: rgba(250, 250, 250, 100);
}
QSpinBox:disabled {
    background-color: rgba(255, 255, 255, 100);
    color: rgba(25, 25, 25, 100);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="header_colorMarks" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QWidget {
    
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(55, 71, 79, 255), stop:0.5 rgba(38, 50, 56, 255), stop:1 rgba(34, 42, 48, 255));
border-top: 1px solid #455a64;
border-bottom: 1px solid #191919;
}

QLineEdit {
    background-color: #455a64;
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(109, 138, 158, 100), stop:0.805 rgba(69, 89, 99, 126), stop:1 rgba(36, 49, 56, 255));
    border-radius: 10px;
    border: 1px solid #191919;
   color: #78909c;
padding:0px 6px 2px 14px;
font: 11px &quot;Open Sans&quot;;
	image: url(:/textures/Common/Tournament/icon_search.png);
    image-position: right bottom;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>18</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QWidget" name="container_bttn_newMark" native="true">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>21</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">.QWidget {
    background-color: #455a64;
    border-radius: 4px;
border-bottom: 1px solid #445862;
}
QAbstractButton {
    border: 1px solid #191919;
border-radius: 3px;
font: 14px &quot;Open Sans Semibold&quot;;
color: #fafafa;
}</string>
        </property>
        <widget class="QPushButton" name="bttn_newMark">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>88</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>20</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>@Tooltip_notes_bttn_NewLabel_82</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(34, 43, 49, 255), stop:1 rgba(69, 89, 99, 255));
border: 1px solid #191919;
border-radius: 3px;
font: 11px &quot;Open Sans &quot;;
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(44, 53, 59, 255), stop:1 rgba(69, 99, 109, 255));
}

QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(41, 51, 54, 255), stop:0.5 rgba(28, 30, 36, 255), stop:1 rgba(34, 35, 42, 255));
padding-top: 1px;
color: rgba(250, 250, 250, 100);
}
QPushButton:disabled {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(34, 43, 49, 100), stop:1 rgba(69, 89, 99, 100));
color: rgba(250, 250, 250, 100);
}
EntityWidget {
	background-color: rgba(255, 255, 255, 250);
}
EntityWidget:disabled {
	background-color: rgba(255, 255, 255, 100);
}</string>
         </property>
         <property name="text">
          <string>@bttn_newMark</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QWidget" name="container_bttns_arrows" native="true">
        <property name="minimumSize">
         <size>
          <width>68</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>68</width>
          <height>21</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">.QWidget {
    background-color: #455a64;
    border-radius: 4px;
border-bottom: 1px solid #445862;
}

QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #222a30, stop:1 #37474f);
font: 11px &quot;Open Sans &quot;;

}


QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #222a30, stop:0.8 #37474f, stop:1 #455a64);
}

QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:1 #222a30, stop:0 #37474f);
padding-top: 1px;
}
QPushButton:disabled {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(34, 43, 49, 100), stop:1 rgba(69, 89, 99, 100));
color: rgba(250, 250, 250, 100);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="bttn_up">
           <property name="minimumSize">
            <size>
             <width>34</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>34</width>
             <height>20</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>@Tooltip_notes_bttn_up_83</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
border: 1px solid #191919;
border-radius: 0px;
border-top-left-radius: 3px;
border-bottom-left-radius: 3px;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/textures/Common/icons/settings/arrow_up.png</normaloff>:/textures/Common/icons/settings/arrow_up.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bttn_down">
           <property name="minimumSize">
            <size>
             <width>34</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>34</width>
             <height>20</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>@Tooltip_notes_bttn_down_84</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
border: 1px solid #191919;
border-left: 1px solid #455a64;
border-radius: 0px;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/textures/Common/icons/settings/arrow_down.png</normaloff>:/textures/Common/icons/settings/arrow_down.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MarkersTableView" name="listView_markers">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>0</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>38</red>
           <green>50</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>38</red>
           <green>50</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>38</red>
           <green>50</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>38</red>
           <green>50</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>38</red>
           <green>50</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>38</red>
           <green>50</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>38</red>
           <green>50</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>38</red>
           <green>50</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>38</red>
           <green>50</green>
           <blue>56</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">QAbstractItemView {
    background-color: #263238;
    outline:0;
    border: none;
    font: 16px &quot;Open Sans&quot;;
border-top: 1px solid #455a64;
}

QAbstractItemView::item {
    height: 38px;
    background-color: #263238;
    color: #fafafa;
    outline:0;
    padding: -1px -2px -1px 16px;
}
QAbstractItemView::text {
    left: 6px;
}

QAbstractItemView::item:alternate {
    background-color: #37474f;
}
QAbstractItemView::item:selected {
    background-color: #cfd8dc;
    color: #191919;
}
QAbstractItemView::item:hover {
    /*background-color: #acb6ba;*/
}

QAbstractItemView::icon {

width: 27px;
padding-right: 10px;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="autoScrollMargin">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>260</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>36</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>12</number>
     </attribute>
    </widget>
   </item>
  </layout>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionload">
   <property name="text">
    <string>load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MarkersTableView</class>
   <extends>QTableView</extends>
   <header>markerslist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
